packages/notebook/src/widget.ts

```diff
@@ -190,6 +191,7 @@ export class StaticNotebook extends Widget {
     this.node.dataset[UNDOER] = 'true';
     this.node.dataset[CODE_RUNNER] = 'true';
     this.rendermime = options.rendermime;
+    this.translator = options.translator || nullTranslator;
```

Comment:
You should pass the translator when creating the widget in `packages/notebook/src/widgetfactory.ts#L98` - you can get it within the factory as `this.translator`.

You should prefer the nullish operator to the or operator:

```suggestion
    this.translator = options.translator ?? nullTranslator;
```